app/code/Magento/Tax/Model/Config.php

```diff
@@ -832,20 +832,20 @@ public function getInfoUrl($store = null)
      * If it necessary will be returned conversion type (minus or plus)
      *
      * @param null|int|string|Store $store
-     * @return bool
+     * @return int
```

Comment:
Currently, the method still returns int and bool value because `$this->displayCartPricesBoth()` returns bool.
Please add bool return type to doc block.